generated/nidcpower/session.py

```diff
@@ -2855,6 +2873,35 @@ def _get_error(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=True)
         return int(code_ctype.value), description_ctype.value.decode(self._encoding)
 
+    def _lock_session(self):
```

Comment:
It's up to you, but if you're hand-coding both _lock_session() and lock(), I think it just complicates things to have them be separate.

Reply:
Moved lock into lock_session template